### PR TITLE
Use TensorFlow ops for Keras LearningRateSchedule

### PR DESCRIPTION
**Pull request by haoyuz**
Create a piecewise learning rate schedule with warmup class and use it for in-graph learning rate computation. This change overlaps the learning rate calculation and assignment with other function execution and reduce the gaps between steps due to slow callbacks.

### CODE REVIEW

**Comment by haoyuz** [comment]
Did a full convergence test with `use_tensor_lr` and got 76.2% at 90th epoch.

File: official/resnet/keras/keras_imagenet_main.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 LR_SCHEDULE = [    # (multiplier, epoch to start) tuples
     (1.0, 5), (0.1, 30), (0.01, 60), (0.001, 80)
 ]
+TRAIN_EPOCH_SIZE = 1281167
```
[the remark this answers is not in the document]
Comment (haoyuz) [reply official/resnet/keras/keras_imagenet_main.py]: Fixed. Thanks!
